### ISSUE TITLE
ModuleNotFoundError: No module named 'jax.numpy'

### ISSUE THREAD

**Issue by vballoli** [deepmind/chex#226]
On installing the latest version of Chex directly from GitHub, the following error pops up when importing chex:

```
File ".../anaconda3/lib/python3.8/site-packages/chex/__init__.py", line 17, in <module>
    from chex._src.asserts import assert_axis_dimension
  File "...anaconda3/lib/python3.8/site-packages/chex/_src/asserts.py", line 26, in <module>
    from chex._src import asserts_internal as _ai
  File ".../anaconda3/lib/python3.8/site-packages/chex/_src/asserts_internal.py", line 34, in <module>
    from chex._src import pytypes
  File ".../lib/python3.8/site-packages/chex/_src/pytypes.py", line 19, in <module>
    import jax.numpy as jnp
ModuleNotFoundError: No module named 'jax.numpy'
```

Jax: v0.3.34
jaxlib: v0.1.69

**Comment by vballoli** [comment]
Hi @hbq1 , really sorry about the jax version, the 3 was a typo. Yep, my conda installation had installed a previous version, it works fine with the latest release.

<img width="399" alt="jax" src="https://user-images.githubusercontent.com/26684166/212694213-04c2574f-3cd2-4ba3-b33b-1b680ccefb3a.PNG">

On a different note, this import `import jax.numpy as jnp` breaks in previous versions(before <0.4.1) - is there a need to update the [requirements.txt](https://github.com/deepmind/chex/blob/master/requirements/requirements.txt) (I can open a PR)

**Comment by hbq1** [comment]
I see, thanks for confirming! 

> import jax.numpy as jnp breaks in previous versions(before <0.4.1) 
This is quite odd... I think I've been doing `jax.numpy as jnp` since I heard about JAX :P 
Might it be possible that something got incorrectly installed in your environment?

**Comment by hbq1** [comment]
Chex successfully installs with jax==0.3.14 as well

**Comment by vballoli** [comment]
Yeah, I verified it in a separate colab installation. I believe it's a WSL installation issue(jax.numpy wasn't being recognized), probably a silent jax error on WSL that I missed.

Thank you so much for your time on this @hbq1, I appreciate it!
